### PR TITLE
Removes rlp-browser

### PR DESCRIPTION
**Pull request by douglance**
TODO: 

- [x] Run all test suites

### CODE REVIEW

File: src/ethereum.ts [hunk 1]
```diff
@@ -7,8 +7,7 @@ import cbor from 'borc';
 //@ts-expect-error - This third-party package is not typed properly
 import { TypedDataUtils } from 'eth-eip712-util-browser';
 import { keccak256 } from 'js-sha3';
-import { encode as rlpEncode } from 'rlp';
-import rlp from 'rlp-browser'; // [TODO: Deprecate]
+import rlp from 'rlp';
```
Comment (alex-miller-0) [inline src/ethereum.ts:10]: Pls import just the encode function as before.

Comment (douglance) [reply src/ethereum.ts]: 7ce1ed8

**Review by KevinVitale — APPROVED** [review]

File: src/ethereum.ts [hunk 2]
```diff
@@ -416,14 +415,14 @@ const buildEthRawTx = function (tx, sig, address) {
...
-  let rlpEncodedWithSig = rlp.encode(newRawTx);
+  let rlpEncodedWithSig = rlpEncode(newRawTx);
   if (tx.type) {
     rlpEncodedWithSig = Buffer.concat([
       Buffer.from([tx.type]),
       rlpEncodedWithSig,
     ]);
   }
-  return { rawTx: rlpEncodedWithSig.toString('hex'), sigWithV: newSig };
+  return { rawTx: Buffer.from(rlpEncodedWithSig).toString('hex'), sigWithV: newSig };
```
Comment (alex-miller-0) [inline src/ethereum.ts:425]: What is `sigWithV`?

**Review by alex-miller-0 — COMMENTED** [review]
Looks fine, but you're throwing a new param in a response that isn't documented or used in any tests. Can you clarify what that is?

File: src/ethereum.ts (hunk 2 above)
Comment (alex-miller-0) [reply src/ethereum.ts:425]: Omg I'm sorry I'm blind

**Review by alex-miller-0 — APPROVED** [review]
Sorry sorry I misread something. This looks good!

**Pull request merged**